releases/3.1.0-beta.2.md

```diff
@@ -0,0 +1,60 @@
+---
+title: TiDB 3.1 Beta.2 Release Notes
+category: Releases
+---
+
+# TiDB 3.1 Beta.2 Release Notes
+
+发版日期：2020 年 3 月 9 日
+
+TiDB 版本：3.1.0-beta.2
+
+TiDB Ansible 版本：3.1.0-beta.2
+
+## TiDB
+
+- 支持在列属性上添加 `AutoRandom` 关键字，控制系统自动为主键分配随机整数，避免 `AUTO_INCREMENT` 自增主键带来的写入热点问题 [#14555](http://github.com/pingcap/tidb/pull/14555)
+- 新增通过 DDL 语句为表创建、删除列存储副本的功能 [#14537](https://github.com/pingcap/tidb/pull/14537)
+- 新增优化器可自主选择不同的存储引擎的功能 [#14537](https://github.com/pingcap/tidb/pull/14537)
+- 新增 SQL Hint 支持不同的存储存储引擎的功能 [#14537](https://github.com/pingcap/tidb/pull/14537)
+- 新增通过 `tidb_replic_read` 系统变量从 Follower 上读取数据的功能 [#13464](https://github.com/pingcap/tidb/pull/13464)
+
+## TiKV
+
++ Raftstore
+    - 新增 `peer_address` 参数，为其他类型的服务提供通过不同端连接此 TiKV server 的能力 [#6491](https://github.com/tikv/tikv/pull/6491)
+    - 修复静默 Region 读数据处理不当导致无法处理读请求的问题 [#6450](https://github.com/tikv/tikv/pull/6450)
+    - 新增 `read_index` 和 `read_index_resp` 监控项，用于监控 `ReadIndex` 请求数 [#6610](https://github.com/tikv/tikv/pull/6610)
+    - 修复 `ReadIndex` 在 leader 切换时可能导致系统 panic 的问题 [#6613](https://github.com/tikv/tikv/pull/6613)
+    - 修复 Hibernate Region 在某些特殊条件下未被正确唤醒的问题 [#6730](https://github.com/tikv/tikv/pull/6730) [#6737](https://github.com/tikv/tikv/pull/6737) [#6972](https://github.com/tikv/tikv/pull/6972)
++ PD Client
+    - 新增将本地线程统计信息汇报给 PD 的功能 [#6605](https://github.com/tikv/tikv/pull/6605)
++ Backup
+    - 用 Rust 的 `async-speed-limit` 流控库替代 `RocksIOLimiter` 流控库，避免备份时拷贝多次内存的问题 [#6462](https://github.com/tikv/tikv/pull/6462)
+    - 修复备份数据时备份了多余的数据，导致恢复数据时数据索引不一致的问题 [#6659](https://github.com/tikv/tikv/pull/6659)
+    - 修复备份时因处理已被删除的值逻辑不正确导致系统 panic 的问题 [#6726](https://github.com/tikv/tikv/pull/6726)
+
+## PD
+
+- 新增 location label 的名字中允许使用斜杠 `/` 的功能 [#2084](https://github.com/pingcap/pd/pull/2084)
+- 修复因 rule checker 给 Region 分配 store 失败导致系统 panic 的问题 [#2161](https://github.com/pingcap/pd/pull/2161)
+
+## TiFlash
+
+- 初始版本
+
+## Tools
+
++ TiDB Lightning
+    - 优化配置项，部分配置项在没有进行配置的时候使用[文档](/reference/tools/tidb-lightning/config.md)中的默认配置 [#255](https://github.com/pingcap/tidb-lightning/pull/255)
```

Comment:
Is it better to use a clear link text that describes the content of this document?
```suggestion
    - 优化配置项，部分配置项在没有进行配置的时候使用[TiDB Lightning 配置参数](/reference/tools/tidb-lightning/config.md)中的默认配置 [#255](https://github.com/pingcap/tidb-lightning/pull/255)
```

Reply:
Yes, it is.